examples/flava/data/datamodules.py

```diff
@@ -141,36 +142,38 @@ def __init__(
     ):
         super().__init__()
         self.train_dataset_infos = train_infos
+        self.text_columns = text_columns
         self.val_dataset_infos = val_infos
         if self.val_dataset_infos is None:
             self.val_dataset_infos = train_infos
         self.tokenizer = tokenizer
+        if self.tokenizer is None:
+            self.tokenizer = BertTokenizer.from_pretrained(TEXT_DEFAULT_TOKENIZER)
```

Comment:
Usually stuff like this should happen in setup?

Reply:
yeah i was trying to reuse it in MLMDatamodule but reverted it in favor of adding a setup to the MLM DM